x-pack/plugins/lens/public/app_plugin/shared/edit_on_the_fly/lens_configuration_flyout.tsx

```diff
@@ -61,6 +61,8 @@ export function LensEditConfigurationFlyout({
   navigateToLensEditor,
   displayFlyoutHeader,
   canEditTextBasedQuery,
+  isNewPanel,
+  onDeletePanel,
```

Comment:
nit: I prefer `deletePanel` since calling this function actually initiates the deletion. To me, `onDeletePanel` implies that this is a routine that should run during panel deletion, but doesn't cause the deletion itself. 🤷‍♂️

Reply:
Sure, done!